<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addwindow</class>
 <widget class="QMainWindow" name="addwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>员工后台管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="photo.qrc">
    <normaloff>:/2.png</normaloff>:/2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#widget
{
	background-color: rgba(0, 0, 0,0.5);
	border:none;
	border-radius:10px;
}
QMainWindow#addwindow
{
border-image: url(:/5.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>主菜单</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>9</y>
      <width>300</width>
      <height>360</height>
     </rect>
    </property>
    <widget class="QPushButton" name="addAllButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>310</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>261</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="addNamelabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>姓名    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addNameEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>姓名</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="addGenderlabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>性别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addGenderEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>男/女</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="addAgelabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>年龄</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addAgeEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>0-150</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="addTimelabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>入职时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addTimeEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>年/月/日</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="addStatelabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>在职状态</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addStateEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>在职/离职</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="addQuitTimelabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>离职时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addQuitTimeEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>年/月/日</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="addCalllabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>手机号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addCallEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>11位号码</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="addCarlabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>车牌号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addCarEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>#X#####</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="addAllButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="girllabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="boylabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/4.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="RollText" name="widgetRoll" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>791</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollText</class>
   <extends>QWidget</extends>
   <header location="global">rolltext.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
